cdn-load-balancer/android/ExoPlayerCDNLoadBalancerPlugin/app/src/main/res/xml/network_security_config.xml

```diff
@@ -3,7 +3,6 @@
     <base-config cleartextTrafficPermitted="true">
         <trust-anchors>
             <certificates src="system" />
-            <certificates src="user" />
```

Comment:
Technically, this was there for customers that use a proxy in the middle, so that we accept to SSL handshake with their proxy once they trust it in their phone settings. I guess it's better to remove it and add something to the README.

Reply:
Hum, I think this was removed when I unified the resources across all sample apps, so that's probably because it was missing in the project I copied the resource from 😅 

I'll try to add it everywhere

Reply:
Addressed in [9d17f23](https://github.com/streamroot/streamroot-samples/pull/44/commits/9d17f2375b7343955c6bf8f26ce4aee4d9d2d380)